src/umpire/strategy/ZeroByteHandler.cpp

```diff
@@ -0,0 +1,90 @@
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
Looks like you accidentally committed an empty file `src/umpire/strategy/ZeroByteHandler.`

Reply:
Whoops, will fix that.